python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -800,7 +800,7 @@ async def start_profile(
             output_dir=output_dir,
             num_steps=num_steps,
             activities=activities,
-            with_stack=with_stack,
+            with_stack=False,
```

Comment:
does this change still applicable with the current version? also if this is not an urgent fix maybe we should not make this change.